src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java

```diff
@@ -101,7 +102,12 @@ private String generateFullDescription(VacancyDTO dto) {
     }
 
     private ResponseDTO getPage(int offset) {
-        ResponseSpec request = getRequest(String.format(url, offset));
+        ResponseSpec request = getRequest(uriBuilder -> uriBuilder
+                .queryParam("sort", "recent")
+                .queryParam("category[]", "software-development")
```

Comment:
@mighty-mhsl here is example, from code you explicitly know that parser has filter by category and set as software-development.

When from [current version](https://github.com/fakng-agrgtr/fakng-agrgtr/blob/1f46e4de9ae159d9228ca141f04c5c6bf7ef12a6/src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java) you can't figure it out. Besides that there is no mention about it even in comments.

Then the method `getAllVacancies` lying to you. It doesn't. It gives you vacancies from specific category.
https://github.com/fakng-agrgtr/fakng-agrgtr/blob/1f46e4de9ae159d9228ca141f04c5c6bf7ef12a6/src/main/java/com/fakng/fakngagrgtr/parser/amazon/AmazonParser.java#L52